apps/dotcms-ui/src/app/view/components/dot-contentlet-editor/components/dot-reorder-menu/dot-reorder-menu.component.html

```diff
@@ -1,6 +1,6 @@
 <dot-iframe-dialog
     *ngIf="!!url"
-    (close)="onClose($event)"
+    (close)="onClose()"
```

Comment:
shutdown?

Reply:
this overlaps other part of the fix, will be fixed during the merges

Reply:
Rename the event.